firmware/application/apps/ui_text_editor.hpp

```diff
@@ -75,12 +75,20 @@ class TextViewer : public Widget {
     // Gets the length of the current line.
     uint16_t line_length();
 
-   private:
-    static constexpr int8_t char_width = 5;
-    static constexpr int8_t char_height = 8;
+    void text_zoom(Rect parent_rect) {
+        zoom = !zoom;
+        char_height = zoom ? 16 : 8;
```

Comment:
Do the Fonts have these sizes? Not like they'll change, but might be nicer to get the value from the Font, vs hard coding constants.